web/js/index.js

```diff
@@ -22,7 +22,26 @@ const loadTexture = (path, loader, onProgress) => {
 }
 
 // Create scene and camera
-let scene = new THREE.Scene();
+var scene;
+var scene1 = new THREE.Scene();
+var scene2 = new THREE.Scene();
+
+scene = scene1;
+
+//The button loading animation uses Bootstrap and Font Awesome
+$('#myButtonID').on('click', function(event){
```

Comment:
I'd just update the indentation here and change _myButtonID_ to a more descriptive id name 👍